src/preview/rewriteStyleSheet.ts

```diff
@@ -1,9 +1,12 @@
 import { PSEUDO_STATES, EXCLUDED_PSEUDO_ELEMENT_PATTERNS } from "../constants"
 import { splitSelectors } from "./splitSelectors"
 
+const pseudoStateRegExp = (global: boolean, pseudoStates: string[]) =>
+  new RegExp(`(?<!(?:${EXCLUDED_PSEUDO_ELEMENT_PATTERNS.join("|")})\\S*):(${pseudoStates.join("|")})`, global ? "g" : undefined)
 const pseudoStates = Object.values(PSEUDO_STATES)
-const matchOne = new RegExp(`:(${pseudoStates.join("|")})`)
-const matchAll = new RegExp(`:(${pseudoStates.join("|")})`, "g")
+const matchOne = pseudoStateRegExp(false, pseudoStates)
```

Comment:
`matchOne` and `matchAll` should have been ignoring pseudo-states that follow excluded patterns. (Though in practice, I'm not sure it made a difference, since we took those into account during replacement.)